qcodes/plots/pyqtgraph.py

```diff
@@ -4,6 +4,7 @@
 import numpy as np
 import pyqtgraph as pg
 import pyqtgraph.multiprocess as pgmp
+from qtpy import QtWidgets
```

Comment:
@jenshnielsen  actually this broke  everything ! We need to use the on from pyqtgraph